docs/index.rst

```diff
@@ -0,0 +1,13 @@
+Welcome to lts_array's documentation!
+=====================================
+
+.. toctree::
+   :caption: README:
+
+   README.rst
```

Comment:
I think — if you were to go with the strategy of having the README.rst in the top level of the repo — that you could use this and remove `docs/README.rst`. You'd have to test though, I'm not positive
```suggestion
   ../README.rst
```

Reply:
This doesn't work. The in-docs README is a command to `.. include::` the actual README on the level up. Either adding the README on the level up or adding the include statement to the index file results in Sphinx being unable to find the file.

Reply:
Ahh, yes. I forgot that you'd need to use this trick: https://stackoverflow.com/a/17217041

Reply:
Whoops, I'm sorry. Looks like you've already done this